samples/bluetooth/peripheral_nucleo_wb55rg/README.rst

```diff
@@ -0,0 +1,30 @@
+.. _bluetooth-peripheral_nucleo_wb55rg:
```

Comment:
Let's rename sample to st_ble_sensor, as this is actually more linked to the mobile application than the board you've used.

Reply:
done